data/large/ground-truth/synth-CO2-1-3300-3700-cm-1-P0.005-t3000-v-r-mf0.01-p1-sl1.4nm.json

```diff
@@ -1,17 +1,18 @@
 {
-  "fileName": "synth-CO2-1-3300-3700-cm-1-0.005-t3000-v-r-p1-sl1.4nm.spec",
+  "fileName": "synth-CO2-1-3300-3700-cm-1-P0.005-t3000-v-r-mf0.01-p1-sl1.4nm.spec",
   "molecule": "CO2",
   "isotope": "1",
-  "from_wavelength": 3300,
-  "to_wavelength": 3700,
+  "wmin": 3300,
+  "wmax": 3700,
+  "wunit": "cm",
   "pressure": 0.005,
   "mole_fraction": 0.01,
   "path_length": 1,
   "slit": "1.4 nm",
   "fit": {
     "Tgas": 3000,
     "bound_Tgas": [
-      2000,
+      2500,
```

Comment:
Minor suggestion : because theoretically any parameter could be fitted, you may want to use a `PARAM_bounds` syntax for the keys

Or, suggestion 2, which simplifies the interface 
- keep the initial value of Tgas in the upper dictionary.
- your fit dicotio'zry becomes `{Tgas:True} ` for normal fitting, `{Tgas:(min, max)} ` for bounds. Initial value is the one from the dictiokzty above 
-  (nice to have) You could even set-up expressions and formulas. I.e. `{Tgas:'2*Tvib'} ` which would be evaluated dynamically. I've already set up a similar thing in Fitroom. See DynVar there. 
Use

Reply:
Mr. @erwanp , for now I just focus on fitting temperatures instead. Can I test on other parameters in the second phase of this GSOC project?

I will try the formating as you recommended, using regex.

Reply:
Mr. @erwanp , I like your suggestion 2 which truly simplifies the interface. I will implement it right away.
But, I don't think we need to keep the initial value of `Tgas` and other fit params in the upper dictionary, because the user can just state it in `"fit` as `{Tgas : 1200}` for instance. So now the fit param statement becomes:

- If `Tgas` is numerical (for example, `{Tgas : 1200}`) : set it as initial value as apply a default bounding range of +- 500.
- If `Tgas` is list or tuple (for example, `Tgas : [500, 2000]` or `Tgas : (500, 2000)`) : set initial value = (upper - lower) / 2 and apply the stated bounding range.
- If `Tgas` is string (for example, `Tgas : '2*Tvib'`) : apply DynVar.